Makefile

```diff
@@ -25,7 +26,7 @@ WASM_O = ${WASM_LIBS:%=${WASM_OUT}/%.o}
 V8_BUILD = ${V8_ARCH}.${V8_MODE}
 V8_V8 = ${V8_DIR}/v8
 V8_DEPOT_TOOLS = ${V8_DIR}/depot_tools
-V8_PATH = ${V8_DEPOT_TOOLS}:${PATH}
+V8_PATH = ${BASE_DIR}/${V8_DEPOT_TOOLS}:${PATH}
```

Comment:
Instead of introducing BASE_DIR, wouldn't it suffice to simply change this line to

V8_PATH = ${realpath ${V8_DEPOT_TOOLS}}:${PATH}

Reply:
When that variable is evaluated `v8` and `v8/depot_tools` don't exist yet.
```
$ realpath v8
/tmp/wasm-v8/wasm-c-api/v8
```
```
$ realpath v8/depot_tools
realpath: v8/depot_tools: No such file or directory
```
so `${realpath ${V8_DEPOT_TOOLS}}` ends up empty.

Reply:
Then you want abspath, not realpath.

Reply:
I see what you mean
```
V8_PATH = $(abspath ${V8_DEPOT_TOOLS}):${PATH}
```
would work since `abspath` doesn't check whether the directory exists